Resources/Locale/en-US/botany/components/seed-extractor-component.ftl

```diff
@@ -1,3 +1,4 @@
 ## Entity
 
-seed-extractor-component-interact-message = You extract some seeds from the {$name}.
\ No newline at end of file
+seed-extractor-component-interact-message = You extract some seeds from the {$name}.
+seed-extractor-component-no-seeds = The {$name} has no seeds!
```

Comment:
Definite articles like 'the' have a function for them in localization

Reply:
Usage is inconsistent in the existing codebase, so I picked what is done more frequently (explicitly writing out 'the'). A simple search will show that this is the case.

As you may know, there are languages where the definite article (English 'the') must match in number, gender, case, and more with the noun being referred to, e.g. "**the** computer" is "**das** Computer" but (finding something) "by **the** sea" is "an **dem** Meer".

Reply:
If it is more frequent it's because most of the loc files have not been updated. The definite article function is more recently than you'd think and indefinite articles exist now but are new to the point they are almost unused.

Reply:
> Usage is inconsistent in the existing codebase, so I picked what is done more frequently (explicitly writing out 'the'). A simple search will show that this is the case.
> 
> As you may know, there are languages where the definite article (English 'the') must match in number, gender, case, and more with the noun being referred to, e.g. "**the** computer" is "**das** Computer" but (finding something) "by **the** sea" is "an **dem** Meer".

Use the localization function ffs

Reply:
Sure, I'll make the change. I just wanted to make sure there were no major structural issues first.